sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerClient.java

```diff
@@ -633,6 +633,29 @@ public String generateUserDelegationSas(BlobServiceSasSignatureValues blobServic
         return this.client.generateUserDelegationSas(blobServiceSasSignatureValues, userDelegationKey);
     }
 
+    /**
+     * Generates a user delegation SAS for the container using the specified {@link BlobServiceSasSignatureValues}.
+     * <p>See {@link BlobServiceSasSignatureValues} for more information on how to construct a user delegation SAS.</p>
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.storage.blob.BlobContainerClient.generateUserDelegationSas#BlobServiceSasSignatureValues-UserDelegationKey-String-Context}
+     *
+     * @param blobServiceSasSignatureValues {@link BlobServiceSasSignatureValues}
+     * @param userDelegationKey A {@link UserDelegationKey} object used to sign the SAS values.
+     * @see BlobServiceClient#getUserDelegationKey(OffsetDateTime, OffsetDateTime) for more information on how to get a
```

Comment:
Same comment as the other Javadoc with inlined `@see`

Reply:
Fixed